test/backup_test.js

```diff
@@ -433,7 +434,7 @@ describe('Backup', () => {
 
         console.log('Backup test: Ensure that messages.zip exists');
         const zipPath = path.join(backupDir, 'messages.zip');
-        const messageZipExists = fs.existsSync(zipPath);
+        const messageZipExists = fse.existsSync(zipPath);
```

Comment:
As stated above, we could do `fs: require('fs-extra')` and then keep `fs.existsSync` and other APIs.